CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.2.1] - 2017-07-19
```

Comment:
Yeah, I wasn't sure what to do on the versioning in the changelog. Should I just have added my fix to the "Unreleased" section, or should I have not added to this document at all?